### ISSUE TITLE
Compressing to war

### ISSUE THREAD

**Issue by omerman** [lookfirst/gulp-helpers#14]
gulp helpers is lack of `compress` task.
it will be useful to include a `compress` task as well

Cheers

**Comment by zivl** [comment]
you mean compression in general, right?
not `.war` file specifically...

**Comment by omerman** [comment]
Yes

**Comment by zivl** [comment]
Merged pull request #16. 
Usage example:

``` javascript
taskMaker.defineTask('compress', {
    taskName: TASK_NAME, 
    src: SOURCE, 
    filename: FILENAME_WITH_EXTENSION, 
    dest: DEST});
```

**Comment by omerman** [comment]
Thanks!
